### PR TITLE
Moderation and Safety section

### PR DESCRIPTION
**Pull request by SylviaMakuch**
Finished Moderation section.

### CODE REVIEW

File: components/token-talk/TokenTalk.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import styled from "styled-components";
 
 const TokenTalkContainer = styled.div`
-    padding: 75px 0px;
+    padding: 50px   0px;
```
Comment (ChristopherTrimboli) [inline components/token-talk/TokenTalk.tsx]: too many spaces

File: components/token-talk/TokenTalk.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ export default function TokenTalk() {
     return (
         <TokenTalkContainer id="token-talk">
             <TokenTalkHeader>$NUDE and $NUDENFT Tokens</TokenTalkHeader>
-            <TokenTalkText>We will be launching an ERC-20 and ERC-721 token used for in-app transactions, NFT minting, governance, staking, investing and all that good tokenomics sauce we expect from crypto projects.</TokenTalkText>
```
Comment (ChristopherTrimboli) [inline components/token-talk/TokenTalk.tsx]: not sure why `token` was removed

File: components/moderation/config.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        image: approval,
+        backgroundColor: "#CC00FF"
+    },
+    {
+        title: "Reporting and Blocking",
+        text: "To deal with abuse and harrasement, we give you the tools to limit the levels of interaction you feel comfortable with.",
+        image: blockingUser,
+        backgroundColor: "#FF51E3"
+    },
+
```
Comment (ChristopherTrimboli) [inline components/moderation/config.ts]: cleanup some of the blank empty lines in this file

File: components/moderation/ModerationRow.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        <HiddenRow>
+            <ModerationRowContainer backgroundColor={backgroundColor}>
+                <Title>{title}</Title>
+                <Text>{text}</Text>
+            </ModerationRowContainer>
+            <ModImg src={image} />
+        </HiddenRow>
+
+
+
```
Comment (ChristopherTrimboli) [inline components/moderation/ModerationRow.tsx]: cleanup empty lines

File: components/moderation/ModerationRow.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+const Title = styled.h2`
+    color: white;
+    text-align: center;
+    margin-bottom: 10px;
+    font-size: 30px;
+    font-family: Rock Salt, Open Sans;
+        @media screen and (max-width: 600px) {
+        font-size: 24px;
+        }
```
Comment (ChristopherTrimboli) [inline components/moderation/ModerationRow.tsx]: this indentation is janky in the CSS

File: components/moderation/Moderation.tsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+                                text={text} 
+                                backgroundColor={backgroundColor} 
+                                key={index} 
+                            />
+                })
+            }
+            
+        </ModerationContainer>
+    );
+};
```
Comment (ChristopherTrimboli) [inline components/moderation/Moderation.tsx:42]: spaces and indentations in this file cleanup please

**Review by ChristopherTrimboli — CHANGES_REQUESTED** [review]
Some empty lines and in general just need to format code better please, thanks :)

File: components/moderation/Moderation.tsx (hunk 6 above)
Comment (SylviaMakuch) [reply components/moderation/Moderation.tsx:42]: Cleaned up

File: components/moderation/ModerationRow.tsx (hunk 5 above)
Comment (SylviaMakuch) [reply components/moderation/ModerationRow.tsx]: Hopefully you like it now!

File: components/moderation/ModerationRow.tsx (hunk 4 above)
Comment (SylviaMakuch) [reply components/moderation/ModerationRow.tsx]: done

File: components/token-talk/TokenTalk.tsx (hunk 1 above)
Comment (SylviaMakuch) [reply components/token-talk/TokenTalk.tsx]: completed

File: components/token-talk/TokenTalk.tsx (hunk 2 above)
Comment (SylviaMakuch) [reply components/token-talk/TokenTalk.tsx]: put back

File: components/moderation/config.ts (hunk 3 above)
Comment (SylviaMakuch) [reply components/moderation/config.ts]: done

**Pull request closed without merging**